docs/index.md

```diff
@@ -1,17 +1,109 @@
 ---
-title: Tilt API Reference
+title: Tilt API Server Reference
 layout: api
 ---
 
-Tilt is a toolkit for building multi-service dev environments.
+If you're trying to set up Tilt for the first time, you're in the wrong place!
+But we'll help you find where you should be.
 
-Tilt offers many first-party data types like Docker builds and Kubernetes
-appliers. But long-term, we want to make it easier for users to define their own
-data types and behaviors.
+## For First-Time Tilt Users
 
-The future of Tilt is a simple model consisting of very few types of
-building blocks, and a mix of uniformity and versatility whereby using the same
-simple elements one can build complex systems and different types of
-functionality.
+For new users, you want the [Tutorial](https://docs.tilt.dev/tutorial/index.html)
+or one of the example projects.
 
-This page is the reference documentation for Tilt's public API.
+<ul>
+  {% for page in site.data.examples %}
+     <li><a href="https://docs.tilt.dev/{{page.href | escape}}">{{page.title | escape}}</a></li>
+  {% endfor %}
+</ul>
+
+For detailed help writing a Tiltfile, you want the [Tiltfile
+Reference](https://docs.tilt.dev/api.html).
+
+## For Dev Environment Maintainers
+
+This reference site is for users who have already set up Tilt, and are:
+
+- trying to diagnose why Tilt isn't behaving like you expect, or
+
+- writing new templates for dev environments to share across teams, or
+
+- scraping metrics about your dev environment to track internally, or
+
+- contributing to Tilt itself (!)
+
+## How to Explore the API Server on Your Own
+
+First, start up a Tilt environment:
+
+```
+tilt up
+```
+
+Then, in a separate terminal, run:
+
+```
+tilt api-resources
+```
+
+This will print out all the objects that your current environment supports. If
+you're familiar with `kubectl`, this may look familiar!  The Tilt CLI is a thin
+wrapper around `kubectl` for querying the Tilt API Server. It supports many of
+the same verbs and flags.
+
```

Comment:
Are Tilt api-resources custom K8s resources? Does that mean you always have to have a k8s cluster running even if you're using docker-compose?

Reply:
rewrote the section a bit in a way that might help

Reply:
I am still not 100% sure
Does this mean, these are not k8s resources and I don't need a k8s cluster, it just uses the same pattern?
You need to spell it out for me :)

Reply:
ok, i just removed this line, since it seems to be more confusing than helpful